data/avgi/avgi 1 first contact.txt

```diff
@@ -0,0 +1,1055 @@
+# Copyright (c) 2023 by Azure
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+mission "Avgi: Lost in Twilight"
+	invisible
+	landing
+	to complete
+		never
+	on enter
+		system
+			attributes "twilight" "tangled shroud"
+		log "Became trapped in a strange region after entering a system my jump drive could not return from."
+		fail
+		set "avgi: lost in twilight"
+		dialog phrase "lost in twilight"
+
+phrase "lost in twilight"
+	word
+		`As soon as you jump into the system, you notice something is wrong. The characteristic plasma of your jump drive fades unnaturally quickly, and as you check your ship's systems, you notice its temperature is somehow much lower than normal. You attempt to plot a jump back to where you came from, but your computer tells you the system is out of range. It appears you are stuck in this region for now.`
+
+mission "Avgi: First Contact"
+	autosave
+	name "An Alien Encounter"
+	description "Accompany the alien ship to the destination marked on your map."
+	landing
+	source
+		attributes "avgi"
+	destination "Peripheria"
+	mark "Arche"
+	mark "Ballad"
+	clearance
+	on offer
+		set "avgi: aberrant siege"
+		event "twilight label"
+		event "tangled shroud label"
+		conversation
+			branch aviskir
+				has "flagship planet: Aviskir"
+			branch coreworld
+				has "flagship planet attribute: avgi core"
+				not "flagship planet attribute: station"
```

Comment:
There are only two planets with "avgi core", Aviskir and Livolua, and neither of them have "station".
You're already branching away for Aviskir, so this branch is basically just `has "flagship planet: Livolua"`. Are you expecting the results of these filters at the time of this mission being offered to be different later? If not, it would be much clearer to readers to just use Livolua here.